pkg/jobconfig/files.go

```diff
@@ -610,32 +611,45 @@ func MakeRegexFilenameLabel(possibleRegex string) string {
 	return label
 }
 
-// IsGenerated returns true if the job was generated using prowgen
-func IsGenerated(job prowconfig.JobBase) bool {
-	_, generated := job.Labels[LabelGenerated]
-	return generated
+// IsGenerated returns true if the job was generated according to ANY of the generatedLabels
+func IsGenerated(job prowconfig.JobBase, generatedLabels ...string) bool {
+	for _, generatedLabel := range generatedLabels {
+		_, generated := job.Labels[generatedLabel]
```

Comment:
`if _, generated := job.Labels[generatedLabel]; generated {
	return true
}`

Reply:
cool, I didn't realize you could use that syntactic sugar if the function returned more than one value.

Reply:
I think there are a couple other places in your changes where you're doing similar checks that could be updated with this.